src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,38 @@
 
 class Animal {
   // write your code here
+  constructor(name, health = 100) {
```

Comment:
`health` should not be passed to the constructor as it is always 100 when animal is created